src/Evolution/Systems/Cce/Initialize/ZeroNonSmooth.hpp

```diff
@@ -51,14 +51,14 @@ struct ZeroNonSmooth : InitializeJ {
         "Number of linearized inversion iterations."};
     static type lower_bound() noexcept { return 10; }
     static type upper_bound() noexcept { return 1000; }
-    static type default_value() noexcept { return 300; }
+    static type suggested_value() noexcept { return 300; }
   };
 
   struct RequireConvergence {
     using type = bool;
     static constexpr Options::String help = {
         "If true, initialization will error if it hits MaxIterations"};
-    static type default_value() noexcept { return false; }
+    static type suggested_value() noexcept { return false; }
```

Comment:
Same for this one, I'd prefer it to stay a `suggested_value`, but for the value to now be `true`